docs/operator-docs/provider.md

```diff
@@ -0,0 +1,77 @@
+---
+id: provider
+title: Provider Configuration
+hide_title: true
+---
+
+# Provider Configuration
+
+In Cumulus a Provider represents a endpoint from which data is ingested.   For example, a http server serving data from a data provider, or an S3 bucket deployed within your organization with data staged for ingest into Cumulus.
+
+Please note:
+
+* While *connection* configuration is defined here, things that are more specific to a specific ingest setup (e.g. 'What target directory should we be pulling from' or 'How is duplicate handling configured?') are generally defined in a Rule or Collection, not the Provider.
```

Comment:
Going back and forth but I think it would be nice if "rule" and "collection" linked to the page describing them

Reply:
I think I agree, but those pages don't exist in the Operator docs yet (and writing them feels ... out of scope?) , and otherwise the explicit description only exists in the Cookbooks Intro page.... which we're linking to from the Glossary. 

But the entire point of this ticket is that the Cookbook oriented example isn't a comprehensive configuration document, and we appear to need that given user/operator feedback on that point. 

What do you think if I write the ticket for Rules/Collections like this one and include cross linking in the AC?    Else write a stub that refers back to the Cookbook intro?   

🤔

Reply:
I guess I was just thinking of linking to the cookbook intro page for now. But given the additional complexity I think we can skip it for now.

I really think the definition of rules/providers/collections needs to be done much more uip front in our docs, like a "Getting started" section, but that's definitely out of scope